components/Card.js

```diff
@@ -81,13 +81,12 @@ const CardDropdownItem = styled.a`
   }
 
   &:not(:last-child) {
-    border-bottom: 1px solid #e8e3e3;
+    border-bottom: 1px solid ${props => props.theme.grays.platinum};
   }
 `;
 
 const CardAction = styled.a`
-  ${fontSize} color: #20588f;
```

Comment:
Just to be sure: Should this color be removed?

Reply:
Yes. The color of anchor tags is defined in the global styles, so this is unnecessary duplication